src/test/java/com/booleanuk/extension/Extension3Test.java

```diff
@@ -6,7 +6,15 @@
 public class Extension3Test {
     @Test
     public void shouldPass() {
-        Poker poker = new Poker();
-        Assertions.assertEquals("Replace this method with your own methods and tests", poker.extensionThreeMethods());
-    }
+    Poker poker = new Poker();
+
+    String[] hand1 = {"A", "A", "A", "A", "2"};  // Replace with actual hands
+    String[] hand2 = {"K", "K", "K", "K", "3"};  // Replace with actual hands
+
+    // Call the method with the actual hands
+    String result = poker.extensionThreeMethods(hand1, hand2);
+
+    // Update the expected result based on your implementation
+        Assertions.assertEquals("Hand 1 wins", result);
+}
 }
```

Comment:
The use of a separate helper class CardValueConverter for converting card values is a neat approach. It helps in keeping your main class uncluttered and focuses on the core logic. You could also consider using a HashMap in your main Poker class for the same purpose. But great work.